### PR TITLE
feat(DDI-1278): Full width page elements adjustments

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

**Comment by Radha-Alberta** [comment]
- [ ] For full width, there should be 24px spacing around the content of the notification banner. Currently it is 72px left and right. (Image 1)

#### Image 1 
image

**Comment by Radha-Alberta** [comment]
- [ ] Angular and React code samples are giving error (Image 1)

#### Image 1
image
image

**Review by vanessatran-ddi — DISMISSED** [review]

File: libs/web-components/src/components/page-block/PageBlock.svelte [hunk 1]
```diff
@@ -52,6 +52,16 @@
...
+  }
+  @media (min-width: 640px) {
+    .page-content {
+      padding: 0 2rem;
+    }
+  }
+  @media (min-width: 1024px) {
+    .page-content {
+      padding: 0 4.5rem;
+    }
```
Comment (chrisolsen) [inline libs/web-components/src/components/page-block/PageBlock.svelte:65]: Should padding be changed or should the max-width be changed instead.

File: libs/web-components/src/components/hero-banner/HeroBanner.svelte [hunk 2]
```diff
@@ -47,6 +49,10 @@
     width: 100%;
     padding: 3.5rem 0;
   }
+  .content-container {
```
Comment (chrisolsen) [inline libs/web-components/src/components/hero-banner/HeroBanner.svelte:52]: These classes are trying to change how the component works, is it not possible to have the component do this instead?

**Review by chrisolsen — APPROVED** [review]

**Review by vanessatran-ddi — APPROVED** [review]

**Pull request merged**